addons/default/visiosoft/base-theme/resources/scss/style.scss

```diff
@@ -0,0 +1,2771 @@
+      }
+    }
+  }
+}
+
+#urun-detay {
+  .slick-initialized .slick-slide {
+    display: flex;
+    justify-content: center;
+  }
+}
+
+.heightSpecial_1 {
+  height: calc(100% - 55px);
+}
+
+.zoomImage {
+  position: fixed;
+  width: 100% !important;
+  height: 100vh !important;
+  z-index: 999999999999999999;
+  top: 0;
+  right: 0;
+
+  .tab-pane.active {
+    background-color: #4c4c4c !important;
+  }
+}
+
+.dontShow {
+  display: none !important;
+}
+
+.logo-proje {
+  img {
+    width: 110px;
+  }
+}
+
+
+.border-none-2 {
+  @media screen and (max-width: 1200px) {
+    border: none !important;
+  }
+}
+
+.placeholder-color {
+  &::placeholder {
+    color: #b6b8b9;
+  }
+}
+
+
+
+#project {
+  .burger {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    display: flex;
+    top: 0 ;
+    bottom: 0 ;
+    z-index: 1000;
+    left: -100%;
+  }
+
+  .burger-side.show{
+    display: flex;
+    left: 0 ;
+  }
+
+  .burger-other.show {
+    left: 0 ;
+  }
+
+  .burger.show {
+    display: flex;
+    left: 0 ;
+  }
+
+
+  .burger-side {
+    top: 0 ;
+    height: 100%;
+    width: 60%;
+    left: -100%;
+    color: #434343;
+    font-weight: 500;
+    background-color: white;
+    z-index: 300;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    transition: left .4s ease-in-out .25s;
+  }
+
+  .burger-other {
+    width:100%;
+    height: 100%;
+    background-color: rgba(0,0,0,0.2);
+    position: absolute;
+    left: -100%;
+    transition: all 0s ease;
+  }
+
+
+  .burger-header-area {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 5px;
+    transition: background-color 0.5s ease;
+    border-radius: 50%;
+    svg {
+      fill: white;
+    }
+  }
+
+  .burger-header-area:hover {
+    background-color: rgb(207, 207, 207);
+    opacity: .6;
+  }
+
+}
+
+.img-area-proje {
+  height: 450px;
+  border: 1px solid grey;
+}
+
+.data-option {
+  width: 117px;
+  transition: all .2s linear;
+  .data-option-img {
+    transition: all .3s linear;
+    i {
+      font-size: 23px;
+      height: 28px;
+      transition: all .2s linear;
+    }
+  }
+  &:hover {
+    border: 1px solid #e0241e !important;
+    color: #e0241e !important;
+    .data-option-img {
+      background-color: #e0241e !important;
+      i {
+        color: white !important;
+        font-size: 28px !important;
+      }
+    }
+  }
+}
+
+.border-special {
+  border: 1px solid #fda657;
+  .gift-icon {
+    width: 57px;
+    height: 78px;
+    @include flex(center, center, row);
+    background-color: #fda657;
+    i {
+      font-size: 20px;
+    }
+  }
+}
+
+.mb-40 {
+  margin-bottom: 40px;
+}
+
+.fw-500 {
+  font-weight: 500;
+}
+
+.fw-600 {
+  font-weight: 600;
+}
+
+.fw-700 {
+  font-weight: 700;
+}
+
+.w-90 {
+  width: 90%;
+}
+
+.fs-20 {
+  font-size: 20px;
+}
+
+.fs-24 {
+  font-size: 24px;
+}
+
+.fs-28 {
+  font-size: 28px;
+}
+
+.fs-14 {
+  font-size: 14px !important;
+}
+
+.w-fit {
```

Comment:
It exists in post.css why do we repeat it? all custom classes can be in a single file

Reply:
These are already existing files in hepsiemlak repo. I just copied them in base-theme. As @fatihalp is aware of the situation, we have not much time for configuration. Correct me if I am wrong, please.